packages/samples/headless-commerce-react/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "headless-commerce-react",
```

Comment:
can you put `@coveo/headless-commerce-react-sample` here so it matches other proejcts and can be then started by using the command below ? thanks :) 

`npm run start -w @coveo/headless-commerce-react-sample`